src/app/components/elements/PromoContent.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { IsaacPodDTO } from "../../../IsaacApiTypes";
+import { apiHelper } from "../../services";
+import { Col, Row } from "reactstrap";
+import { IsaacContentValueOrChildren } from "../content/IsaacContentValueOrChildren";
+import { Tabs } from "./Tabs";
+
+export const PromoContent = ({ item }: { item: IsaacPodDTO }) => {
```

Comment:
I know that there are some tests in `IsaacApp.test.tsx`, but this new component is missing some unit tests.

Reply:
Unit tests added for this